src/server/dom.ts

```diff
@@ -28,11 +28,12 @@ import { FatalDOMError, RetargetableDOMError } from './common/domErrors';
 export class FrameExecutionContext extends js.ExecutionContext {
   readonly frame: frames.Frame;
   private _injectedScriptPromise?: Promise<js.JSHandle>;
-  private _debugScriptPromise?: Promise<js.JSHandle | undefined>;
+  readonly world: types.World|null;
```

Comment:
Shouldn't eslint complain about missing spaces around `|`?

Reply:
Apparently there is no rule that covers this in eslint. [space-infix-ops](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/space-infix-ops.md) looked promising, but I upgraded our typescript-eslint package and tried it and it didn't work.